install/kubernetes/cilium/templates/cilium-agent-clusterrolebinding.yaml

```diff
@@ -9,6 +9,6 @@ roleRef:
   name: cilium
 subjects:
 - kind: ServiceAccount
-  name: cilium
+  name: {{ .Values.serviceAccounts.cilium.name | quote }}
```

Comment:
This might be one of the inconsistencies that you mentioned, so I do find it a bit weird, that this service account, we always use the parameterized `name` value here, but for the other service accounts, `name` is hard-coded if `create=false`.

I believe we should always use the `name` variable in the role bindings and serviceAccountName field on the pods, regardless of the value of `create`.

Reply:
Noted, definitely make sense.